_includes/payment-url.md

```diff
@@ -29,3 +30,46 @@ URL of the Payment stored.
+doing a full browser (top frame) redirect out of the Seamless View. 3-D Secure
+requires this, for example.
+
+For mobile flows, some payment instruments work best when app-to-app switching
+is enabled and handled automatically (Swish, Vipps etc). To solve this, it is
+important that the third party app or site understand where to redirect the
+consumer back to after the flow on their end is completed.
+
+The `paymentUrl` is the URL Swedbank Pay will provide to the third party for
+handling the redirect back to your site or app. When receiving the consumer back
+on the `paymentUrl` either in an app or web page it is important to restore the
+Seamless View to let the payment finalize the flow. For in-app it is important
+that you either implement the `onPaymentCompleted` event or let the Seamless
+View redirect to the `completeUrl` before intercepting the WebView. If you
+intercept the WebView when the consumer device is redirected to the `paymentUrl`
+it can lead to issues. If you want to handle payment errors in your own code,
+you should also subscribe to other events provided by the Seamless View
+JavaScript and shut down the Seamless View if any of these events occur.
+
+Events to subscribe to for full control over the payment flow are:
+
+{:.table .table-striped}
+| Event                        | Description                                                                                                                                         |
+| :--------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `onError`                    | Will trigger any time a system error occur from the JavaScript.                                                                                     |
+| `onPaymentFailed`            | will call registrered endpoint if provided. If not subscribed to will redirect to conpleteUrl                                                       |
+| `onPaymentCanceled`          | will be called if the payment is aborted by the consumer or triggered by you with the Abort command on the payment                                  |
+| `onPaymentTransactionFailed` | this will trigger if one payment attempt failed. Standard functionality are to let the consumer be able to retry the payment from the payment page. |
+
+When implementing the Seamless View flow into a WebView in your mobile app, you
+should use a [custom scheme][custom-scheme] or [Universal Link][universal-link]
+in the `paymentUrl` for handling automatic switching between your app and the
+payment app on the mobile device.
+
+[custom-scheme]: https://developer.apple.com/documentation/uikit/inter-process_communication/allowing_apps_and_websites_to_link_to_your_content/defining_a_custom_url_scheme_for_your_app
+[universal-link]: https://developer.apple.com/ios/universal-links/
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[final-definition] Move definitions to the end of the file (after the node at line `75`)](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5356607)